includes/UserRightsProxy.php

```diff
@@ -233,6 +233,8 @@ function invalidateCache() {
 			array( 'user_id' => $this->id ),
 			__METHOD__ );
 
+		$this->db->insert( 'user_replicate_queue', [ 'user_id' => $this->id ] );
```

Comment:
Are we sure this will always be shared DB?

Reply:
I didn't really check but the `app` version of MW has `user` table only on shared DB, so the call above wouldn't make sense in the context of a different one anyway